samples/drivers/led_tlc59731/src/main.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2024 Javad Rahimipetroudi <javad.rahimipetroudi@mind.be>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/drivers/led.h>
+#include <zephyr/drivers/led_strip.h>
+
+LOG_MODULE_REGISTER(tlc59731_, LOG_LEVEL_DBG);
+
+#define TLC59731_HIGH       1
+#define TLC59731_TEST_DELAY 1000 /* ms */
+
+#define TLC59731_STRIP_NUM_PIXELS 1
+
```

Comment:
with https://github.com/zephyrproject-rtos/zephyr/pull/70023 this separate sample shouldn't be needed